build/musicbrainz/scripts/start.sh

```diff
@@ -9,11 +9,12 @@ then
   /musicbrainz-server/script/compile_resources.sh
 fi
 
+dockerize -wait tcp://db:5432 -timeout 60s -wait tcp://mq:5672 -timeout 60s -wait tcp://redis:6379 -timeout 60s /start_mb_renderer.pl
+
 if [ -f /crons.conf -a -s /crons.conf ]
 then
   crontab /crons.conf
   cron -f &
 fi
 
-/start_mb_renderer.pl
 start_server --port=5000 -- plackup -I lib -s Starlet -E deployment --nproc 10 --pid fcgi.pid
```

Comment:
That follows https://github.com/metabrainz/musicbrainz-docker/pull/123#discussion_r379899209, yes about moving `dockerize` only.



 `start_server` and `cron` stuff should not be executed at all if dockerize failed for some reason.

Reply:
So it should be enough if we do ``set -e``? Or do you want to go with ``set -eou pipefail`` to also catch errors in pipes and unset variables?

Reply:
I see no pipe here, otherwise yes, `set -e -u` should do.